### PR TITLE
Add option --with-libxml2 which is needed for DASH support

### PR DESCRIPTION
**Pull request by Brmm**
libxml2 is needed for DASH support

### CODE REVIEW

File: Formula/ffmpeg.rb [hunk 1]
```diff
@@ -82,6 +83,7 @@ class Ffmpeg < Formula
   depends_on "xvid" => :optional
   depends_on "zeromq" => :optional
   depends_on "zimg" => :optional
+  depends_on "libxml2" => :optional
```
Comment (retokromer) [inline Formula/ffmpeg.rb]: This should go before (alphabetically), otherwise LGTM.

**Review by retokromer — CHANGES_REQUESTED** [review]

**Comment by slhck** [comment]
I would actually prefer to have this as a default, seeing that DASH encoding is becoming more and more important. What do you think?

**Review by retokromer — APPROVED** [review]

**Comment by retokromer** [comment]
For me it’s almost indifferent if optional or default, with a very little preference for optional.

**Comment by slhck** [comment]
Merged manually via https://github.com/varenc/homebrew-ffmpeg/commit/0a74c531bf67c4fdd3dbb1401aaecf0bb46505b9 since the alphabetical ordering wasn't correct, so I slightly modified your commit. Thanks!

**Comment by retokromer** [comment]
Alphabetical order is needed in the option, in order to make `brew audit --strict` happy. Yet, of course, it’s better to use always the alphabetical order.

**Comment by retokromer** [comment]
@Brmm Thank you!

**Pull request closed without merging**